### PR TITLE
chore(docs): include `webdriver.Key` and `webdriver.promise` in API page

### LINKED ISSUE #402: docs: Should provide API info on webdriver.FOO interfaces like promises and KEY
Some links to these should be added to https://github.com/angular/protractor/blob/master/docs/api.md

### PR DESCRIPTION
**Pull request by sjelin**
Closes #402

### CODE REVIEW

**Comment by juliemr** [comment]
There's a bunch of gross stuff at the end of the promise API that users don't care about. Can we do anything about these?
screen shot 2015-11-10 at 8 00 24 am

**Comment by sjelin** [comment]
Made the change we discussed offline.  Let me know what you think.  :hammer: :wrench: :scissors:

File: website/docgen/processors/filter-promise.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+var _ = require('lodash');
+
+/**
+ * Remove docs of private variables from selenium's promise.js file
```
Comment (juliemr) [inline website/docgen/processors/filter-promise.js]: nit: add '.' at end of sentence.

**Comment by juliemr** [comment]
LGTM.

**Pull request merged**